src/switchroot/ostree-prepare-root.c

```diff
@@ -533,7 +533,9 @@ main (int argc, char *argv[])
             }
         }
 #else
-      errx (EXIT_FAILURE, "composefs: enabled at runtime, but support is not compiled in");
+      /* if composefs is configured as "maybe", we should continue */
+      if (composefs_config->enabled == OT_TRISTATE_YES)
```

Comment:
Also, this is a perfectly fine fix, but...it may actually be more readable to move the `#ifdefs` around such that we do it more like:

```
#ifdef HAVE_COMPOSEFS
  if (composefs_config->enabled != OT_TRISTATE_NO)
    {
    }
#else
  if (composefs_config->enabled == OT_TRISTATE_YES)
    errx (...)
#endif
```
or so